editor.planx.uk/src/pages/FlowEditor/components/Sidebar/Search/facets.ts

```diff
@@ -26,10 +26,110 @@ const drawBoundaryData: SearchFacets = [
 ];
 
 /** Data fields used across PlanX components */
-export const DATA_FACETS = [
+export const DATA_FACETS: SearchFacets = [
   ...generalData,
   ...fileUploadAndLabelData,
   ...calculateData,
   ...listData,
   ...drawBoundaryData,
 ];
+
+const basicFields: SearchFacets = [
+  "data.text",
+  "data.title",
+  "data.description",
+];
+
+const moreInformation: SearchFacets = [
+  "data.notes",
+  "data.howMeasured",
+  "data.policyRef",
+  "data.info",
+];
+
+const checklist: SearchFacets = ["data.categories.title"];
+
+const nextSteps: SearchFacets = [
+  "data.steps.title",
+  "data.steps.description",
+  "data.steps.url",
+];
+
+const fileUploadAndLabel: SearchFacets = [
+  "data.fileTypes.name",
+  "data.fileTypes.notes",
+  "data.fileTypes.howMeasured",
+  "data.fileTypes.policyRef",
+  "data.fileTypes.info",
+];
+
+/** List, Page, and MapAndLabel components share this structure */
+const schemaComponents: SearchFacets = [
+  "data.schema.fields.data.title",
+  "data.schema.fields.data.description",
+  "data.schema.fields.data.options.description",
+  "data.schema.fields.data.options.text",
+  "data.schema.fields.data.options.data.val",
```

Comment:
nit: I think `data.schema.fields.data.options.data.val` should be considered a data field only search term, rather than general search?

Reply:
Fantastic catch thank you! 🦅 👀  Looks like a copy/paste error